hedging/migrations/20221013061329_add-okex-transfers.up.sql

```diff
@@ -3,10 +3,9 @@ CREATE TABLE okex_transfers (
   client_transfer_id VARCHAR(32) PRIMARY KEY,
   correlation_id UUID NOT NULL,
   
-  action VARCHAR(20) NOT NULL CHECK (action in ('withdraw-all', 'withdraw', 'deposit')),
-  transfer_type VARCHAR(20) NOT NULL CHECK (transfer_type in ('internal', 'external')),
+  action VARCHAR(32) NOT NULL CHECK (action in ('transfer-trading-to-funding', 'transfer-funding-to-trading', 'deposit', 'withdraw')),
 
-  currency VARCHAR(20) NOT NULL,
+  currency VARCHAR(16) NOT NULL,
```

Comment:
I wouldn't use `VARCHAR(x)`... if it turns our the assumption of `x` is wrong its tricky to change. There are no performance benefits over `VARCHAR`.